packages/kmod-5.10-nvidia/kmod-5.10-nvidia.spec

```diff
@@ -136,20 +136,115 @@ install kernel/nvidia-peermem/nvidia-peermem.o %{buildroot}%{_cross_datadir}/nvi
 install kernel/nvidia-drm.mod.o %{buildroot}/%{_cross_datadir}/nvidia/tesla/%{nvidia_tesla_470_version}/module-objects.d
 install kernel/nvidia-drm.o %{buildroot}/%{_cross_datadir}/nvidia/tesla/%{nvidia_tesla_470_version}/module-objects.d
 
+# Binaries
 install -m 755 nvidia-smi %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{nvidia_tesla_470_version}
 install -m 755 nvidia-debugdump %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{nvidia_tesla_470_version}
 install -m 755 nvidia-cuda-mps-control %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{nvidia_tesla_470_version}
 install -m 755 nvidia-cuda-mps-server %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{nvidia_tesla_470_version}
+
+# We only install the libraries required by all the DRIVER_CAPABILITIES, described here:
```

Comment:
How do we map "utility" to the actual library list?
* libnvidia-ml.so
* libnvidia-cfg.so

Reply:
`libnvidia-container` categorizes the libraries in the groups I used,  you can see it here:

https://github.com/NVIDIA/libnvidia-container/blob/d48f9b0d505fca0aff7c88cee790f9c56aa1b851/src/nvc_info.c#L61

Reply:
Might be good to include this link in the comment as well!